test/test_postman.py

```diff
@@ -6,35 +6,52 @@
 
 from test.settings import L1_URL, GATEWAY_URL
 from test.util import call, deploy, invoke, run_devnet_in_background, load_file_content
-from test.web3_util import web3_call, web3_transact
+from test.web3_util import web3_call, web3_compile, web3_deploy, web3_transact
 
 import atexit
 import time
 import json
 import subprocess
 import requests
 import pytest
+import os
 
-STARKNET_MESSAGING_PATH = "build/contracts/MockStarknetMessaging.json"
-L1L2_EXAMPLE_ETH_PATH = "build/contracts/L1L2Example.json"
+from solcx import install_solc, install_solc_pragma, set_solc_version
+from web3 import Web3
+from web3.contract import Contract
+from semantic_version import Version
```

Comment:
What's this?

Reply:
Was a dependency along with solcx to specify the version of the compiler, but it is not needed as we switched to the `npx hardhat compile` approach